apps/web/src/screens/Profile/modals/ReceiveToken/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { ModalConfigs } from '@walless/app';
+import { Stack } from '@walless/ui';
+
+import ModalHeader from '../components/ModalHeader';
+import ModalWrapper from '../components/ModalWrapper';
+
+import QRCode from './components/QRCode';
+import WalletAddress from './components/WalletAddress';
+
+const ReceiveTokenScreen: FC<{ config: ModalConfigs }> = ({ config }) => {
+	return (
+		<ModalWrapper>
+			<Stack minHeight="70vh">
```

Comment:
avoid using `vh` and `vw` because these units work unpredictably on mobile

Reply:
what should i use instead of "vh" in this case?

Reply:
> I guess paddingTop or paddingSomething with fixed value like `60` if we just want to separate the Modal from screen edges. Not sure I understand the situation hehe.

i got u, sir, thank uuu